packages/app/ui/buttons/RadioButton.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import { cva, cx } from "class-variance-authority";
+
+interface RadioButtonProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  id: string;
+}
+
+export const radioButtonStyles = cva(
+  [
+    "flex items-center justify-center",
+    "cursor-pointer select-none font-medium",
+    "active:ring-4",
+    "disabled:bg-surface-75 disabled:text-em-disabled disabled:cursor-not-allowed disabled:ring-0",
+    "focus:outline-none focus:ring-4",
+    "rounded-full px-3 py-2 space-x-1.5 text-xs",
```

Comment:
This is just a suggestion. 

I would prefer to move this string up to the second element, near the `flex items-center`.
I like to have the padding above.

Also `text-xs` could be near `font-medium` as they are relatable as well.

Not important, just sharing a preference.

Reply:
Maybe we should have some guidelines to better define how to organize classes.

Reply:
Maybe we should. We can also try it loose for a couple more weeks and then decide the guidelines.